kratos/python_interface/kratos_globals.py

```diff
@@ -81,4 +81,21 @@ def HasVariable(self,VarName):
         else:
             return False
 
+    def GetConstitutiveLaw(self, ConstitutiveLawName):
+
+        kernel = self.Kernel
+
+        if kernel.HasConstitutiveLaw(ConstitutiveLawName):
+            return kernel.GetConstitutiveLaw(ConstitutiveLawName)
+        else:
+            raise ValueError("\nKernel.GetConstitutiveLaw() ERROR: ConstitutiveLaw {0} is unknown. Maybe you need to import the application where it is defined?\n".format(ConstitutiveLawName))
```

Comment:
i would say that you already are throwing this error in the c++ side so it will never arrive here

Reply:
`kernel.HasConstitutiveLaw(ConstitutiveLawName)` returns true or false, so it might actually get there

I made it consistent with the GetVariable mehtog, but I can also call `kernel.GetConstitutiveLaw(ConstitutiveLawName)` right away and throw the error in C

Then I can also print the list of registered laws there, I like it

Will do this, thx for the comment